meson.build

```diff
@@ -490,7 +490,7 @@ if get_option('build_backends')
     cuda_cc = get_option('cc_cuda') # Unfortunately option cuda_cc is reserved.
     nvcc_extra_args = []
     if cuda_cc != ''
-      nvcc_extra_args = ['-arch=compute_' + cuda_cc, '-code=sm_' + cuda_cc]
+      nvcc_extra_args = ['--use_fast_math', '-arch=compute_' + cuda_cc, '-code=sm_' + cuda_cc]
```

Comment:
Note: we had a report that this PR is faster without this change on the dev-public channel.

Reply:
that's strange. At my end I am not seeing any perf difference. I have reverted this part to avoid confusion/corner cases.